modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala

```diff
@@ -38,56 +38,45 @@ class ObserveCommandRoutes[F[_]: Async: Compression](
   private val commandServices: HttpRoutes[F] = HttpRoutes.of[F] {
     case req @ POST -> Root / ObsIdVar(obsId) / ClientIDVar(clientId) / "start" /
         ObserverVar(obs) :? OptionalRunOverride(runOverride) =>
-      ssoClient.require(req) { user =>
+      ssoClient.require(req): user =>
         oe.start(obsId, user, obs, clientId, runOverride.getOrElse(RunOverride.Default)) *>
           NoContent()
-      }
 
     case req @ POST -> Root / ObsIdVar(oid) / StepIdVar(step) / ClientIDVar(clientId) / "execute" /
         ResourceVar(resource) / ObserverVar(obs) =>
-      ssoClient.require(req) { user =>
-        oe.configSystem(oid, obs, user, step, resource, clientId) *>
-          NoContent()
-      }
+      ssoClient.require(req): user =>
+        oe.configSystem(oid, obs, user, step, resource, clientId) *> NoContent()
+
+    case req @ POST -> Root / ObsIdVar(obsId) / StepIdVar(stepId) / "startFrom" /
```

Comment:
It seems we have used `obsid/stepid/clientid/<verb>` format in other endpoints, should we make them match?

Reply:
Sure, I'll rearrange